web/components/manalinks/create-links-button.tsx

```diff
@@ -72,6 +73,33 @@ function CreateManalinkForm(props: {
     message: '',
   })
 
+  const expireOptions = {
+    day: 'day',
+    week: 'week',
+    month: 'month',
+    never: 'never',
+  }
+
+  function getExpireTime(timeDelta: string) {
```

Comment:
You could make this more concise with something like:
`return timeDelta === 'never' ? null : dayjs().add(1, timeDelta)`

Reply:
But actually - since it seems like setNewManalink is going to want a time in Ms anyways, we might as well express it in a time in ms; then we can simplify the below code to
```
const expiresTime = timeDelta === 'never' ? null : dayjs().add(1, timeDelta).valueOf()
setNewManalink((m) => { ...m, expiresTime})
```